src/extension/extension.ts

```diff
@@ -353,6 +356,27 @@ export class RunmeExtension {
       context.subscriptions.push(new CloudAuthProvider(context))
     }
 
+    kernel
+      .runProgram('echo $0')
+      .then(({ output }) => {
+        const supportedShells = ['zsh', 'bash']
+        const isSupported = supportedShells.some((sh) => output?.includes(sh))
+
+        if (!isSupported) {
+          const showMore = 'Show more'
+          return window.showWarningMessage('Unsupported shell', showMore).then((answer) => {
```

Comment:
Is it an error or a warning? We can recover when somebody's running `pwsh` or `cmd.exe` or `fish`. Smells like error to me.